def test_create_comment():
    pass


def test_create_comment_not_authorized():
    pass


def test_returns_error_if_create_comment_exception():
    pass


def test_update_comment_by_id():
    pass


def test_update_comment_by_id_invalid_id():
    pass


def test_update_comment_by_id_user_is_not_comment_author():
    pass


def test_update_comment_by_id_not_authorized():
    pass


def test_returns_error_if_update_comment_by_id_exception():
    pass


def test_update_comments():
    pass


def test_update_comments_invalid_ids():
    pass


def test_update_comments_user_is_not_comments_author():
    # will it save the ones that they are the author to?
    # and not do anything about the rest?
    # or not save anything at all?
    pass


def test_update_comments_not_authorized():
    pass


def test_returns_error_if_update_comments_exception():
    # this kinda overlaps on the above
    # since known errors still appear as exceptions I think
    pass
